docs/agent-install-standalone-linux.md

```diff
@@ -0,0 +1,115 @@
+# Install to Linux using a GZIP File
+
+Install the Smart Agent to Linux host using a compressed
+standalone package in a GZIP file.
+
+## Prerequisites
+
+* Kernel version 2.6.32 or higher
+* cap_dac_read_search and cap_sys_ptrace capabilities
```

Comment:
These aren't prerequisites, they are capabilities you can add to the binary after you install it.

Reply:
These are Linux per-thread capabilities. When does the user need to add them to the Smart Agent binary?

Although the [Privileges](https://docs.signalfx.com/en/latest/integrations/agent/advanced-install-options.html#privileges) section describes the capabilities, it's not clear to me exactly when the user needs to add them to the Smart Agent binary.

Please clarify.